### PR TITLE
Added forms to blog app

### PR DESCRIPTION
**Pull request by tjay1760**
In this pull request I:

- Created a method called current_user in your ApplicationController to make current user data available to all controllers.
 It returns the first user from the database.
- Created forms to perform the following functions:
- Created a Post on behalf of the current_user (use the method that you created in your ApplicationController).
- Created a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allowed Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
+    @post = @user.posts.build(post_params)
+    if @post.save
+      redirect_to user_post_path(@user, @post), notice: "Post was created for #{@user.name}"
+
+    else
+      puts @post.errors.full_messages
+      flash.now[:alert] = 'Post could not be created. Please try again.'
+      render :new
+    end
+  end
```
Comment (julie-ify) [inline app/controllers/posts_controller.rb]: - [ ] Posts should be created using the `current_user` you defined in the application controller.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @tjay1760,

Good job so far!

## Highlights!
- Good code organization and working forms ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+    @post = @user.posts.build(post_params)
+    if @post.save
+      redirect_to user_post_path(current_user, @post), notice: 'Post was created'
+
+    else
+      puts @post.errors.full_messages
+      flash.now[:alert] = 'Post could not be created. Please try again.'
+      render :new
+    end
+  end
```
Comment (levy002) [inline app/controllers/posts_controller.rb]: - [ ] As requested by the previous reviewer, kindly make sure to use the `current_user` while creating the new post in order to match the project requirements.

```suggestion
  def create
    @user = current_user
    @post = @user.posts.build(post_params)
    if @post.save
      redirect_to user_post_path(current_user, @post), notice: 'Post was created'

    else
      puts @post.errors.full_messages
      flash.now[:alert] = 'Post could not be created. Please try again.'
      render :new
    end
  end
```

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @tjay1760   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @tjay1760  , 

Great job so far!  

May I highlight the following: 
- [x]  Well done updating your controller as requested👏  ✔️ 
- [x] Your code is clean and well organized✔️



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**